### PR TITLE
Wip sockets

### PR DESCRIPTION
**Pull request by klassenjo**

### CODE REVIEW

File: packages/game/src/lib/player.ts [hunk 1]
```diff
@@ -1,35 +1,23 @@
...
-  public removeCard(card: Card) {
-    const idx = this.#hand.findIndex(
-      (_card) => _card?.name?.toLowerCase() == card.name.toLowerCase(),
-    );
+  const temp: Hand = [...hand];
+  temp[idx] = undefined;
 
-    this.#hand[idx] = undefined;
-  }
+  return temp;
```
Comment (jakeklassen) [inline packages/game/src/lib/player.ts]: Maybe we call this `newHand` instead of `temp`

Comment (klassenjo) [reply packages/game/src/lib/player.ts]: Fixed.

File: packages/backend-socketio/src/schemas/game-data-schema.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import * as TripleTriad from '@tripletriad/game';
+import { Board, Card } from '@tripletriad/game/src/lib/common-types';
+import { toZod } from 'tozod';
+import { z } from 'zod';
+
+export const PlayerLabelSchema = z.enum([
+  TripleTriad.PlayerLabel.One,
+  TripleTriad.PlayerLabel.Two,
+]);
```
Comment (jakeklassen) [inline packages/backend-socketio/src/schemas/game-data-schema.ts]: Looks like zod recommends native enums here: https://zod.dev/?id=native-enums

Comment (jakeklassen) [reply packages/backend-socketio/src/schemas/game-data-schema.ts]: We could always move away from enums too, I think they're not a create idea in TS.

**Pull request merged**